manifests/y/Youdao/YoudaoDict/9.1.5.0/Youdao.YoudaoDict.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Youdao.YoudaoDict
+PackageVersion: 9.1.5.0
+PackageLocale: en-US
+Publisher: 网易公司
+# PublisherUrl:
```

Comment:
```suggestion
PublisherUrl: https://cidian.youdao.com/
```

Reply:
If these changes are applied, I think the local file should be named as zh-cn.

Reply:
I don't see a problem having it with the en-US locale, but having a zh-CN locale as well would be preferable.

Reply:
I mean, if I rename the en-us locale to zh-cn, will it be okay? As I haven't seen an en-us version of youdaodict up till now.

Reply:
Yes, it would be okay but you would have to translate`ShortDescription` for that locale